src/index.ts

```diff
@@ -12,6 +12,24 @@ const checkCommit = async (...refs: string[]) =>
         )
     )
 
+/**
+ * if set, returns CIRCLE_REPOSITORY_MAIN_BRANCH, otherwise origin/master
+ *
+ * This is used for backwards compataibility with repositories that still use master
+ * as their main branch, even though the default has been shifted to 'main' in 2020
+ */
+const getRepositoryRemoteMainBranch = (): string => {
+    const branch = process.env.CIRCLE_REPOSITORY_MAIN_BRANCH
+    if (branch === undefined) {
+        return 'origin/master'
+    }
+    // allows people to make mistakes when setting the main branch
+    if (!branch.startsWith('origin/')) {
```

Comment:
Is there a chance someone uses something different from `origin`? In that case this would fail in a weird way.

Reply:
It's from the perspective of CircleCI. So unless actually has steps to rename the remote in a step, this should be fine